lib/command/src/Obelisk/Command/Deploy.hs

```diff
@@ -150,13 +159,88 @@ deployPush deployPath getNixBuilders = do
 deployUpdate :: MonadObelisk m => FilePath -> m ()
 deployUpdate deployPath = updateThunkToLatest $ deployPath </> "src"
 
+keytoolToAndroidConfig :: KeytoolConfig -> Map Text NValue
+keytoolToAndroidConfig conf = Map.fromList
+  [ ("storeFile", NValue_Path $ _keytoolConfig_keystore conf)
+  , ("storePassword", NValue_Text $ T.pack $ _keytoolConfig_storepass conf)
+  , ("keyAlias", NValue_Text $ T.pack $ _keytoolConfig_alias conf)
+  , ("keyPassword", NValue_Text $ T.pack $ _keytoolConfig_keypass conf)
+  ]
+
+
 deployMobile :: MonadObelisk m => String -> [String] -> m ()
 deployMobile platform mobileArgs = withProjectRoot "." $ \root -> do
   let srcDir = root </> "src"
   exists <- liftIO $ doesDirectoryExist srcDir
   unless exists $ failWith "ob test should be run inside of a deploy directory"
-  result <- nixBuildAttrWithCache srcDir $ platform <> ".frontend"
-  callProcessAndLogOutput (Notice, Error) $ proc (result </> "bin" </> "deploy") mobileArgs
+  when (platform == "android") $ do
```

Comment:
Should platform really be a string here?

Reply:
This now uses a data type